doc/source/conf.py

```diff
@@ -362,4 +362,6 @@ def linkcode_resolve(domain, info):
 myst_enable_extensions = [
     # Enable fieldlist to allow for Field Lists like in rST (e.g., :orphan:)
     "fieldlist",
+    # Enable fencing directives with `:::`
+    "colon_fence",
```

Comment:
We support markdown in our docs! Woah, how did I not know this. :sunglasses:

Reply:
MyST comes with its own syntax quirks but I've started to come around to it. I being a superset of markdown seems like a very strong argument to default to it for new documents. :thinking: